src/shared/modules.mjs

```diff
@@ -289,6 +289,16 @@ const isPolymorphicReducer = compose(
     getPath(['tags'])
 )
 
+const isXUsesMethod = compose(
```

Comment:
We should rename this `isAllowFocusMethod`.

Almost every method is an x-uses method, and this name really confused me when I was reading the code that calls it.

Reply:
Sure, will update

Reply:
done